openapi/Swarm.yaml

```diff
@@ -487,7 +487,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "SwarmCommon.yaml#/components/schemas/Status"
+                $ref: "SwarmCommon.yaml#/components/schemas/HealthStatus"
```

Comment:
This might cause changes on the infra side if they are using this endpoint. Instead I would call the new one BeeStatus or NodeStatus or something.

Reply:
The name is not returned by the endpoint so shouldn't be an issue.